usagov/usagov-2021
# Usagov-1037 elected officials error presentation

## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-1037

## Description

This change updates the wording of the error messages and displays them in the main heading and page intro rather than in the results area.

## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions


* visit `/elected-officials-results`
* edit the page
* add a div with id="resultsSection" around the results portion of the html like this:
```
<div id="resultsSection" class="usa-prose">
    <h2>Your results:</h2>
    <p>Expand the following results and reveal the contact information for the elected officials at each level of government. Then, select their name to see how to contact them and their staff.</p>
    </br>
    <div id="results"></div>
</div>
```
* save the change and view the page
* The page's main heading should say "Invalid address"
* The intro text should say "There was a problem getting results for this address. Please check to be sure you entered a valid U.S. address."
* The results area should not be showing.
* Next visit `/es/funcionarios-electos-resultados`
* edit the page
* add a div with id="resultsSection" around the results portion of the html like this:
```
<div id="resultsSection" class="usa-prose">
    <h2>Sus resultados:</h2>
    <p>Despliegue la siguiente lista de resultados y encuentre la información de contacto de los funcionarios electos en cada nivel del Gobierno. Después, seleccione sus nombres para saber cómo comunicarse con ellos y su personal.</p>
    <div id="results"></div>
</div>
```
* save the change and view the page
* The page's main heading should say "Dirección incorrecta"
* The intro text should say "Tuvimos problemas para obtener resultados con esta dirección. Por favor, verifique si ingresó una dirección válida en EE. UU."
* The results area should not be showing.

### Requires New Config

### Requires New Content

### Validation Steps
- Test instruction 1
- Test instruction 2
- Test instruction 3

## Security Review




## Reviewer Reminders
- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions
Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
